src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -36,23 +36,24 @@ const SettingsAccessLinkManagement = ({
   } = useLinkManagement(enterpriseUUID);
 
   const {
-    customerAgreement: { netDaysUntilExpiration },
+    customerAgreement: { subscriptions },
     couponsData,
```

Comment:
Nit: This variable name is changed in other places, change here too for consistency?